core/src/debug/assets/dummy-mobile-services.json

```diff
@@ -1,23 +1,43 @@
 {
+    "version": "1.0",
+    "clusterName": "192.168.64.74:8443",
+    "namespace": "myproject",
+    "clientId": "example_client_id",
     "services": [
         {
+            "id": "fh-sync-server",
+            "name": "fh-sync-server",
+            "type": "fh-sync-server",
+            "url": "https://www.mocky.io/v2/5a6b59fb31000088191b8ac6",
             "config": {
-                "auth-server-url": "http://keycloak-myproject.192.168.37.1.nip.io/auth",
-                "clientId": "OeFYkwEuUKXvDsSudfXc",
-                "headers": {},
+            }
+        },
+        {
+            "id": "keycloak",
+            "name": "keycloak",
+            "type": "keycloak",
+            "url": "https://www.mocky.io/v2/5a6b59fb31000088191b8ac6",
+            "config": {
+                "auth-server-url": "https://keycloak-myproject.192.168.64.74.nip.io/auth",
+                "clientId": "juYAlRlhTyYYmOyszFa",
```

Comment:
I don't think we need this? Keycloak uses `resource` as defined below for specifying the clientID.